packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -265,13 +265,17 @@ export async function grantRoles(serviceAccountName: string, role: string) {
   return
 }
 
-export async function retrieveCloudSQLConnectionInfo(celoEnv: string, instanceName: string) {
+export async function retrieveCloudSQLConnectionInfo(
+  celoEnv: string,
+  instanceName: string,
+  dbSuffix: string
+) {
   await validateExistingCloudSQLInstance(instanceName)
   const [blockscoutDBUsername] = await execCmdWithExitOnFailure(
-    `kubectl get secret ${celoEnv}-blockscout -o jsonpath='{.data.DATABASE_USER}' -n ${celoEnv} | base64 --decode`
+    `kubectl get secret ${celoEnv}-blockscout${dbSuffix} -o jsonpath='{.data.DATABASE_USER}' -n ${celoEnv} | base64 --decode`
```

Comment:
Could you pull out `${celoEnv}-blockscout${dbSuffix}` into a const so making changes in the future is a little more straightforward?